ebusd-2.1.x/de/vaillant/bai.308523.inc

```diff
@@ -24,6 +24,7 @@ r,,DCRoomthermostat,d.16 Raumthermostat,,,,"0E00",,,onoff,,,Wärmeanforderung vo
 r,,HwcDemand,d.22 WW Anforderung,,,,"5800",,,yesno,,,Brauchwasseranforderung (Zapfung oder Schaltsignal vom Speicher)
 r,,HeatingSwitch,d.23 Winterbetrieb,,,,"F203",,,onoff,,,Wintermodus aktiv (Heizbetrieb und Warmwasser)
 r,,StoragereleaseClock,d.25 Speicherfreigabe,,,,"4704",,,yesno,,,Freigabe der Speichrladung oder des Warmstartspeichers durcheinen eBUS Regler
+r,,PrimaryCircuitFlowrate,d.29 Durchfluss Heizkreis,,,,"FB00",,,uin100,,l/min,Einheit Durchfluss
```

Comment:
are you sure this is valid for all fallback bai units? or which product id does yours have?